app/Http/Controllers/VideosController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Tournament;
+use App\Video;
+use App\Http\Requests;
+use Illuminate\Http\Request;
+
+class VideosController extends Controller
+{
+    /**
+     * Saves new video.
+     * @param Requests\VideoRequest $request
+     * @return redirects
+     */
+    public function store(Requests\VideoRequest $request)
+    {
+        $tournament = Tournament::withTrashed()->findOrFail($request->get('tournament_id'));
+        $this->authorize('logged_in', Tournament::class, $request->user());
+
+        $video = Video::create([
```

Comment:
Please insert some validation here.
- what if the youtube ID provided is non-existent?
- what if the youtube ID is already added (avoid duplicate entries)